twentytwentyone-blocks/experimental-theme.json

```diff
@@ -0,0 +1,203 @@
+{
+	"global": {
+		"settings": {
+			"color": {
+				"gradients": [ ],
+				"palette": [
+					{
+						"slug": "black",
+						"color": "#000000"
+					},
+					{
+						"slug": "dark-gray",
+						"color": "#28303D"
+					},
+					{
+						"slug": "gray",
+						"color": "#39414D"
+					},
+					{
+						"slug": "green",
+						"color": "#D1E4DD"
+					},
+					{
+						"slug": "blue",
+						"color": "#D1DFE4"
+					},
+					{
+						"slug": "purple",
+						"color": "#D1D1E4"
+					},
+					{
+						"slug": "red",
+						"color": "#E4D1D1"
+					},
+					{
+						"slug": "orange",
+						"color": "#E4DAD1"
+					},
+					{
+						"slug": "yellow",
+						"color": "#EEEADD"
+					},
+					{
+						"slug": "white",
+						"color": "#FFFFFF"
+					}
+				]
+			},
+			"typography": {
+				"customFontSize": true,
+				"customLineHeight": true,
+				"fontSizes": [
+					{
+						"slug": "extra-small",
+						"size": "16px"
+					},
+					{
+						"slug": "small",
+						"size": "18px"
+					},
+					{
+						"slug": "normal",
+						"size": "20px"
+					},
+					{
+						"slug": "large",
+						"size": "24px"
+					},
+					{
+						"slug": "extra-large",
+						"size": "40px"
+					},
+					{
+						"slug": "huge",
+						"size": "96px"
+					},
+					{
+						"slug": "gigantic",
+						"size": "144px"
+					}
+				],
+				"spacing": {
+					"customPadding": true,
+					"units": [ "px", "em", "rem", "vh", "vw" ]
+				}
+			},
+			"custom": {
+				"font-primary": "-apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif",
+				"font-weight-light": 300,
+				"color": {
+					"primary": "var(--wp--preset--color--dark-gray)",
```

Comment:
The user can change the presets and consequently change the value of the primary is it expected?

Reply:
For now yes — that's fine. I initially created these custom variables just to make it easier to migrate the existing color rules over from the original version of Twenty Twenty-One. We should be able to remove these and map to the exact preset palette colors instead in a followup PR.